### DISCUSSION TITLE
switch to old display form new '3 - window'

### DISCUSSION THREAD

**Discussion by ASamSam** [WinMerge/winmerge#2007] [General]
Hello, 
sorry, maybe already asked, but
but to switch to the classic merge design from the current new '3-window' display?
i have some habitation and still don`t feel comfortable with the new one...
Sincerely yours, Andrew

**Comment by sdottaka** [comment]
What exactly is the new '3-window' display?
It would be helpful if you could show me a screenshot if possible. If you simply don't want to display files and folders on 3pane, you can display them on 2pane, but that's not what you're talking about, right?

**Reply by tetsuhaut** [reply]
Hi,
I use winmerge as a tool to do `git difftool`. Git commands winmerge to do a 3 ways comparison with (in %USERPROFILE%\\.gitconfig)
~~~
[mergetool "winmerge"]
	cmd = \"C:\\logicielsPortables\\WinMerge\\WinMergeU.exe\" -e -u -dl \"Base\" -dr \"Mine\" \"$LOCAL\" \"$REMOTE\" \"$MERGED\"
~~~
to configure it for a 2 ways comparison, change this line to:
~~~
cmd = \"C:\\logicielsPortables\\WinMerge\\WinMergeU.exe\" -e -u -dl \"Base\" -dr \"Mine\" \"$LOCAL\" \"$REMOTE\"
~~~

**Reply by ASamSam** [reply]
the 'merge' window seems became changed in case of conflict resolving:
before, it was basically left, right panes + result in the bottom

now there are three pane - columns, and seems logic is changed

**Reply by tetsuhaut** [reply]
Dear @ASamSam, the 'merge' window is opened by a command. It is that command that you need to change. have a look at the tool you use to resolve conflits. if this tool is git, the ~/.gitconfig file should contain the command I gave you.

**Reply by ASamSam** [reply]
Hello,  little bit missed with @tetsuhaut answers. I do use winmerge in windows as a separate tool

**Reply by sdottaka** [reply]
@ASamSam
> before, it was basically left, right panes + result in the bottom

WinMerge does not have the function to display the merge result at the bottom. The only tool I know of that does that is TortoiseMerge.
Weren't you using the tool in the image below?

https://tortoisesvn.net/assets/img/TMerge_ThreePane.88b06e39.png

If you are running WinMerge from TortoiseSVN or TortoiseGit, you can switch back to TortoiseMerge from the Diff viewer page in the Settings dialog.

https://tortoisesvn.net/docs/release/TortoiseSVN_en/images/SettingsProgs.png

**Reply by ASamSam** [reply]
Wow, I`m very sorry, seems to be I missed with my habitations. Thank you very much )
